auth/server/base/src/main/java/org/wildfly/security/authz/Roles.java

```diff
@@ -130,6 +132,30 @@ public boolean isEmpty() {
         };
     }
 
+    /**
+     * Construct a new roles collection from a set.
```

Comment:
I would suggest that this method return an immutable view of the roles.

Reply:
@pferraro Thank you, fixed